### DISCUSSION TITLE
Svelte support in Parcel 2

### DISCUSSION THREAD

**Discussion by liegeandlief** [parcel-bundler/parcel#8598] [Q&A] [answered]
Is Svelte supported in Parcel 2? I can't find any documentation relating to it and research that I've done all seems to concern Parcel v1. Would be really nice if this was officially supported like React and Vue.

**Comment by devongovett** [comment, marked as answer]
There's a 3rd party plugin for it. https://github.com/orlov-vo/parcel-transformer-svelte

I agree would be nice to eventually have by default though.

**Reply by mischnic** [reply]
There are some discussions regarding that in this issue: https://github.com/parcel-bundler/parcel/issues/7503
